.github/workflows/main.yml

```diff
@@ -18,6 +18,15 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: '10.x'
-    - run: yarn install
-    - run: yarn test
```

Comment:
We're no longer running this step?

Reply:
Whoops, looks like a copy and paste error from the matrix repo. Fixed in 464c7a2